.env.baklavastaging

```diff
@@ -13,27 +13,27 @@ CLUSTER_DOMAIN_NAME="celo-testnet"
 TESTNET_PROJECT_NAME="celo-testnet"
 
 BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
-BLOCKSCOUT_DOCKER_IMAGE_TAG="72dacea44b1779e5b15069d991453772d4e8f032"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="09add19279a01388295f63df64223264d8a288ea"
 BLOCKSCOUT_WEB_REPLICAS=3
-BLOCKSCOUT_DB_SUFFIX=3
+BLOCKSCOUT_DB_SUFFIX=4
 BLOCKSCOUT_SUBNETWORK_NAME="Baklava Staging"
 
 CELOSTATS_SERVER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-server"
-CELOSTATS_SERVER_DOCKER_IMAGE_TAG="70bbe4d9bd4015ddcb9adfd5ea0e205a043e87c9"
+CELOSTATS_SERVER_DOCKER_IMAGE_TAG="master"
```

Comment:
Is there a reason why we are switching to a master tag here vs. commits? I think for predictable deploys, the latter seem preferrable?

Reply:
Good point. I've changed it to be the latest commit on `master` instead of the literal string `master`